### PR TITLE
OEL-2163: Content news required.

### PR DESCRIPTION
**Pull request by tibi2303**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: oe_showcase.post_update.php [hunk 1]
```diff
@@ -679,3 +679,11 @@ function oe_showcase_post_update_00024(): void {
 function oe_showcase_post_update_00025(): void {
   ConfigImporter::importSingle('profile', 'oe_showcase', '/config/post_updates/00025_event_location', 'core.entity_view_display.node.oe_sc_event.teaser');
 }
+
+/**
+ * News content required.
+ */
+function oe_showcase_post_update_00026(): void {
```
Comment (drishu) [inline oe_showcase.post_update.php]: This is not needed here as we have the oe_starter_content one, we need to rather patch oe_starter_content in composer with a diff, so the new code gets used.

**Review by drishu — CHANGES_REQUESTED** [review]

File: oe_showcase.post_update.php (hunk 1 above)
Comment (tibi2303) [reply oe_showcase.post_update.php]: done.

File: tests/src/ExistingSite/NewsTest.php [hunk 2]
```diff
@@ -71,6 +71,12 @@ public function testCreateNews() {
...
     $this->drupalLogin($user);
 
+    // Assert that the content field is required.
+    $this->drupalGet('node/add/oe_sc_news');
+    $page->fillField('Title', 'Example title');
+    $page->pressButton('Save');
+    $assert_session->pageTextContains('Content field is required.');
+
     // Assert that editors have access to the Simple/Rich text formats.
     $this->drupalGet('node/add/oe_sc_news');
```
Comment (drishu) [inline tests/src/ExistingSite/NewsTest.php]: I'm 90% sure this line is not needed, drupal should stay on the form when it has errors

**Review by drishu — CHANGES_REQUESTED** [review]

**Review by drishu — DISMISSED** [review]

File: tests/src/ExistingSite/NewsTest.php (hunk 2 above)
Comment (tibi2303) [reply tests/src/ExistingSite/NewsTest.php]: done.

**Review by drishu — DISMISSED** [review]

File: oe_showcase.post_update.php [hunk 3]
```diff
@@ -697,3 +697,11 @@ function oe_showcase_post_update_00026(): void {
     $block->save();
   }
 }
+
+/**
+ * News content required.
+ */
+function oe_showcase_post_update_00027(): void {
```
Comment (brummbar) [inline oe_showcase.post_update.php]: Is this needed? We indeed have the one in starter content.

**Review by brummbar — CHANGES_REQUESTED** [review]

File: oe_showcase.post_update.php (hunk 3 above)
Comment (tibi2303) [reply oe_showcase.post_update.php]: removed.

**Review by brummbar — APPROVED** [review]

**Pull request merged**